### PR TITLE
Add the ability to specify not to send certificates with messages

### PR DESCRIPTION
**Pull request by drvespa**
* User reported issue to add ability to not send certificates with #7 
messages. Added new flag -ds to indicate this
* Add unit tests to cover the new scenarios that I added.

### CODE REVIEW

File: MapiSmimeToolkit.cpp [hunk 1]
```diff
@@ -140,7 +142,13 @@ BOOL ParseArgs(int argc, _TCHAR* argv[], ToolkitOptions * pToolkitOptions)
 				pToolkitOptions->bOverWrite = true;
 				break;
 			case 'd':
-				pToolkitOptions->bDefaultSecurityProfule = true;
+				if (_tcsclen(argv[i]) < 3) {
+					pToolkitOptions->bDefaultSecurityProfule = true;
```
Comment (stephenegriffin) [inline MapiSmimeToolkit.cpp:146]: Typo: bDefaultSecurityProfule => bDefaultSecurityProfile

Comment (stephenegriffin) [reply MapiSmimeToolkit.cpp:146]: and really not a fan of just checking length here but I suppose for quick and dirty (very dirty) it kinda works

**Review by stephenegriffin — APPROVED** [review]

File: MapiSmimeToolkit.cpp (hunk 1 above)
Comment (drvespa) [reply MapiSmimeToolkit.cpp]: I fixed the typo and made the parsing of the new switch a little more definitive. I don't like it, but it'll do for now.

**Comment by RufusJWB** [comment]
Thank you for working on this issue! Do you have a ETA for this PR?

**Comment by RufusJWB** [comment]
Did you make progress with this topic?

**Pull request merged**